migrations/migrations/R__Create-Test-Data.sql

```diff
@@ -2285,6 +2311,30 @@ VALUES
   ) ON CONFLICT
 DO NOTHING;
 
+INSERT INTO
+  public.officer (
+    officer_guid,
+    user_id,
+    create_user_id,
+    create_utc_timestamp,
+    update_user_id,
+    update_utc_timestamp,
+    person_guid,
+    office_guid
+  )
+VALUES
+  (
+    '06ff894b-3895-4d32-8a4a-1fcc0be23e47'::uuid,
+    'RRONDEAU',
+    'FLYWAY',
+    '2024-01-22 22:16:16.754',
+    'FLYWAY',
+    '2024-01-22 22:20:48.186',
+    '0667495f-61a5-4d3b-b756-1ee58cb38e23'::uuid,
+    '4a5a94b1-bd47-4611-a577-861d97089903'::uuid
+  ) ON CONFLICT
```

Comment:
As this is the production script for the office_guid you are going to want use the COSHQ guid (c3d8519c-73cb-48a1-8058-358883d5ef4f) alternatively as of release orca you can now leave this value null to indicate you don't have an office (COS Role will act weird though as it expects you to have one)

Also you are going to want to include your auth_user_guid here (77c6040d69b74757903f1cba37404db4)

Reply:
Updated 👍